samples/hello_world/src/main.c

```diff
@@ -5,9 +5,19 @@
  */
 
 #include <stdio.h>
+#include <zephyr/device.h>
```

Comment:
I'm sure you're already on it, but please create separate samples and / or tests